docs/UserGuide.md

```diff
@@ -24,180 +24,180 @@ reliably than traditional GUI based apps.
 Lists all the patients’ names and corresponding patient ID, displaying any tags attached to them and showing the ward they are in.
 
 **Warning: `WARD_NAME` is case sensitive.** `block B ward 2` will refer to a different ward from `Block b Ward 2`
 
-###Format: `lspat [w/WARD_NAME]`
+### Format: `lspat [w/WARD_NAME]`
 `WARD_NAME` can be specified to only list patients in the specified ward. This field is case sensitive. This is an optional parameter.
 
-###Examples:
+### Examples:
 `lspat` will list all patients in each ward on the GUI. Every ward will be displayed in order, with all patients in each ward.\
 `lspat block B ward 2` will display the list of patients that are assigned to block B ward 2 in the GUI
 
 ---
-##Viewing the details of a specific patient: view
+## Viewing the details of a specific patient: view
 Lists the full detail of a specific patient, including their prescriptions/instructions, name, patient ID and tags in the main window.
 
-###Format: `view pid/PATIENT_ID_NUMBER`
+### Format: `view pid/PATIENT_ID_NUMBER`
 
-###Examples:
+### Examples:
 `view pid/A0123456789B` will display all the information associated with the patient with ID number A0123456789B on the GUI.
 
 ---
-##Listing a patient’s prescriptions: lspresc
+## Listing a patient’s prescriptions: lspresc
 Lists the prescriptions and instructions list for a patient in numbered list form for the patient specified by `PATIENT_ID_NUMBER`.
 
-###Format: `lspresc pid/PATIENT_ID_NUMBER`
+### Format: `lspresc pid/PATIENT_ID_NUMBER`
 
-###Examples:
+### Examples:
 `lspresc pid/A0123456789B` will display the list of tasks and prescriptions of patient with ID A0123456789B on the GUI
 
 ---
-##Listing staff members: lsstf
+## Listing staff members: lsstf
 
 Lists staff members’ names and the name of the ward they are assigned to.
 
 **Warning: `WARD_NAME` is case sensitive.** `block B ward 2` will refer to a different ward from `Block b Ward 2`
 
-###Format: `lsstf [w/WARD_NAME]`
+### Format: `lsstf [w/WARD_NAME]`
 `WARD_NAME` can be specified to only list staff in the specified ward. This is an optional parameter.
 
-###Examples:
+### Examples:
 `lsstf` will list all staff assigned to each ward on the GUI. The lists will be grouped according to ward, and each staff
 can appear more than once if they are assigned to more than 1 ward.\
 `lsstf w/block A ward 1` will list all staff assigned to block A ward 1 on the GUI.
 
 ---
-##Listing all wards: lsward
+## Listing all wards: lsward
 Lists all existing wards on the GUI. Only ward names will be displayed.
 
-###Format: `lsward`
+### Format: `lsward`
 
 ---
-##Update patient particulars: editpat
+## Update patient particulars: editpat
 This overwrites the specified patient’s particulars with new particulars. The patient to be edited must exist, i.e. there must exist a
 patient with the given `PATIENT_ID_NUMBER`. **All existing information about the patient will be deleted, and replaced with
 the new information.**
```

Comment:
Shouldn't we only overwrite particulars that are specified?

Reply:
This is for 1.2 where we implement minimum feature set. Will probably have to change the implementation for 1.3 anyway, no need for 1.2 to get too complicated. 

Although if not too troublesome to implement overwrite by necessity the UG can of course be updated to reflect as such. This is up to whoever is implementing this, I came up with this set of functionality as guide only without too much consideration

Reply:
Ok no issue then.